src/components/notification.ts

```diff
@@ -28,7 +30,7 @@ export class Notification {
   private readonly props;
 
   constructor (props: NotificationProps) {
-    this.duration = props.duration !== undefined ? props.duration : 5000;
+    this.duration = props.duration !== undefined ? props.duration : DEFAULT_TIMEOUT;
```

Comment:
nit: `props duration ?? DEFAULT_TIMEOUT` seems easier unless I am missing some logic here

Reply:
Yup, I did not changed the previous code, but sounds a better simplification. I'll include in the next rev.